client/post-editor/editor-term-selector/index.jsx

```diff
@@ -50,23 +54,29 @@ class EditorTermSelector extends Component {
 	}
 
 	render() {
-		const { siteId, taxonomyName } = this.props;
+		const { postType, siteId, taxonomyName } = this.props;
 
 		return (
-			<TermTreeSelector
-				analyticsPrefix="Editor"
-				onChange={ this.boundOnTermsChange }
-				selected={ this.getSelectedTermIds() }
-				taxonomy={ taxonomyName }
-				siteId={ siteId }
-				multiple={ true }
-			/>
+			<div>
+				<TermTreeSelector
+					analyticsPrefix="Editor"
+					onChange={ this.boundOnTermsChange }
+					selected={ this.getSelectedTermIds() }
+					taxonomy={ taxonomyName }
+					siteId={ siteId }
+					multiple={ true }
+				/>
+				<AddTerm taxonomy={ taxonomyName } postType={ postType } />
```

Comment:
Potentially `<AddTerm />` could be used elsewhere, like the menu builder for example - where utilizing that selector will not work.  Right now in the menu builder I think we link back to wp-admin to add a new category.  Might be nice to drop this in there as well.